frontend/src/containers/EditChart.tsx

```diff
@@ -381,6 +385,10 @@ function EditChart() {
             values.chartType === ChartType.DonutChart) && {
             dataLabels: values.dataLabels,
           }),
+          ...((values.chartType === ChartType.PieChart ||
```

Comment:
Same here, is PartToWhole part of this PR?

Reply:
see above